.github/workflows/ci-windows.yml

```diff
@@ -7,7 +7,40 @@ permissions:
 
 jobs:
   Windows:
-    name: Windows
+    name: Windows (Native OPAM)
+    runs-on: windows-latest
+    steps:
+      - name: 🔭 Checkout code
+        uses: actions/checkout@v4
+
+      - name: 😈 Evil hack
+        run: rm rocq-prover.opam
+
+      - name: 😈😈 Evil hack 2
+        run: rm test-suite/success/rewrule.v
+
+      - name: 🐫 Setup OCaml
+        uses: ocaml/setup-ocaml@v3
+        with:
+          ocaml-compiler: 4.14
+          dune-cache: true
+          opam-local-packages: coqide-server.opam rocq-*.opam
```

Comment:
This is broken, the glob format doesn't like this, not that it matters in this case, but we should figure out what the right syntax is.